src/open_inwoner/js/components/anchor-menu/anchor-menu.js

```diff
@@ -1,21 +1,15 @@
 import Gumshoe from 'gumshoejs'
 
-const htmx = (window.htmx = require('htmx.org'))
-
 export class CreateGumshoe {
-  static selector = '.anchor-menu'
+  static selector = '.anchor-menu--desktop a'
 
-  constructor() {
-    new Gumshoe('.anchor-menu--desktop a', {
+  constructor(node) {
+    new Gumshoe(CreateGumshoe.selector, {
       navClass: 'anchor-menu__list-item--active',
       offset: 30,
     })
   }
 }
 
-htmx.onLoad(() => {
-  document.body.addEventListener('htmx:afterSwap', (event) => {
-    const anchors = document.querySelectorAll(CreateGumshoe.selector)
-    ;[...anchors].forEach((anchor) => new CreateGumshoe(anchor))
-  })
-})
+const anchors = document.querySelectorAll(CreateGumshoe.selector)
+;[...anchors].forEach((anchor) => new CreateGumshoe(anchor))
```

Comment:
I created Taiga ticket #1511 to do something about these self-hydrating components.